org.eclipse.jdt.ls.target/org.eclipse.jdt.ls.tp.target

```diff
@@ -33,13 +33,13 @@
             <repository location="http://download.eclipse.org/eclipse/updates/4.10-I-builds/"/>
         </location>
         <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
-            <unit id="org.eclipse.lsp4j.sdk.feature.group" version="0.5.0.v20180903-1212"/>
-            <repository location="http://download.eclipse.org/lsp4j/updates/releases/0.5.0/"/>
+            <unit id="org.eclipse.lsp4j.sdk.feature.group" version="0.6.0.v20181008-1501"/>
+            <repository location="http://services.typefox.io/open-source/jenkins/job/lsp4j/job/master/lastStableBuild/artifact/build/p2-repository/"/>
```

Comment:
http://services.typefox.io/open-source/jenkins/job/lsp4j/job/master/31/artifact/build/p2-repository/  will be a little more stable (not by much), until 0.6.0 gets a milestone